pkgs/servers/home-assistant/default.nix

```diff
@@ -798,6 +796,7 @@ in with py.pkgs; buildPythonApplication rec {
   '';
 
   passthru = {
+    inherit availableComponents;
```

Comment:
What do we expose this for?

Reply:
Hmm, we could extend the update script to handle component tests automatically. :thinking:

Reply:
> What do we expose this for?

It's used in the NixOS module to compare against the components mentioned in the config.